cypress/s3_monitor.py

```diff
@@ -0,0 +1,184 @@
+import boto3
+import json
+import quopri
+import time
+import re
+import argparse
+from pathlib import Path
+
+
+class S3Monitor:
+
+    MAILBOX_BUCKET = "opg-lpa-casper-mailbox"
+    ACTIVATION_EMAILS_PATH = (Path(__file__).parent / "activation_emails").resolve()
+
+    def __init__(self, options):
+        """
+        :param options: dict with v (verbose) and c (is in CI) boolean properties
+        """
+        self.verbose = options["v"]
+        self.in_ci = options["c"]
+        self.s3Client = self.assume_role_and_get_client()
+
+    def assume_role_and_get_client(self):
+        sts = boto3.client(
+            "sts",
+            region_name="eu-west-1",
+        )
+
+        if self.in_ci:
+            print("S3Monitor starting, Assuming CI role")
+            role_arn = "arn:aws:iam::050256574573:role/opg-lpa-ci"
```

Comment:
ensure this is not hard coded here, as i think you might want to run this on the preproduction also.

Reply:
Where would the correct place be that I could get the strings for these roles from? Somewhere in terraform? Note that this is a conditional which switches between roles (and not something I touched), but you're right that it should be fixed.